### ISSUE TITLE
Error in external command after config update

### ISSUE THREAD

**Issue by dodadew** [TheGreatCodeholio/fd_notify_extension#1]
Setting up on Ubuntu 22.04.1 LTS
After running icad_shell.py to update the config file for the external commands and running fd-tone-notify, this error presents itself when a call comes in:
```Sep-17-2022 16:56:45 error :   EXTERN fd_extension: Error running External Command command. Error: spawn cd ENOENT
Sep-17-2022 16:56:45 error :   EXTERN fd_extension: This is an ENOENT error. Typically that means the command is invalid or the path to the command is invalid. Check your path and make sure you can run the command in CMD/Terminal exactly as it appears in the config file.
Sep-17-2022 16:56:45 error : External Command Error: cd /software/testing-fd-tone-notify-extension/ && python3 pre_record.py "[detectorName]" [custom]
Sep-17-2022 16:56:45 debug : Error: spawn cd ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

The change happened here:  #https://github.com/TheGreatCodeholio/fd_notify_extension/commit/0ef78775e2283bc6c862fcfbf4e95174046896fb in lib/config_editor_handler.py
```diff
-     "command": "python3 " + config.fd_tone_notify_extension_path + "post_record.py [timestamp] \"[detectorName]\" [recordingRelPath] [custom]",
+     "command": "cd " + config.fd_tone_notify_extension_path + " && python3 post_record.py [timestamp] \"[detectorName]\" [recordingRelPath] [custom]",
```
If I revert the config external command line to before the commit, it works.  What I am breaking by switching it back, or better yet, what did I miss on why it was switched in the first place?

**Comment by TheGreatCodeholio** [comment]
What happened here was I forgot to fix that part after I had already fixed other portions of the file. I have just committed a change that fixes it.

When I first wrote it I assumed I could run the following "cd /home/icad/fd_notify_extension && python3 post_record.py [data] [data]"

However the way commands are handled with fd-notify it doesn't allow for that to work. it interprets it as separate commands. so you change director, then start a new session and run the python command. So I switched it to run the command directly using a fd_notify_extension path. :)

[Fixing Commit](https://github.com/TheGreatCodeholio/fd_notify_extension/commit/161f05275590bb7951b0f21b33dbda436b7eb472)
